lms/static/scripts/frontend_apps/components/dashboard/OrderableActivityTable.tsx

```diff
@@ -19,33 +23,14 @@ export type OrderableActivityTableProps<T> = Pick<
   navigateOnConfirmRow?: (row: T) => string;
 };
 
-/**
- * List of columns which should start sorted in descending order.
- *
- * This is because for numeric columns ("annotations" and "replies") users will
- * usually want to see the higher values first.
- * Similarly, for date columns ("last_activity") users will want to see most
- * recent values first.
- */
-const descendingOrderColumns: readonly string[] = [
-  // Annotation metrics
-  'last_activity',
-  'annotations',
-  'replies',
-
-  // Course metrics
-  'last_launched',
-  'assignments',
-];
```

Comment:
This was coupling this component with every individual activity table, so I have moved the logic there.